config/build_image-v3.0.0.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
+# This script comprises everything up to and including the boilerplate
+# backing image at version image-v2.0.0. It is used when performing a
+# full build in the appsre pipeline, but is bypassed during presubmit CI
+# in prow to make testing faster there. As such, there is a (very small)
+# possibility of those behaving slightly differently.
```

Comment:
Please edit the corresponding comment in build_image-v2.0.0.sh to look like the one at the top of build_image-v1.0.0.sh (i.e. the bit about "unused, kept for posterity")